cmd/cloud/dbt.go

```diff
@@ -0,0 +1,154 @@
+package cloud
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	cloud "github.com/astronomer/astro-cli/cloud/deploy"
+	"github.com/astronomer/astro-cli/cloud/deployment"
+	"github.com/astronomer/astro-cli/config"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	mountPath      string
+	dbtProjectPath string
+
+	DeployBundle = cloud.DeployBundle
+)
+
+const (
+	dbtDeployExample = `
+Specify the ID of the Deployment on Astronomer you would like to deploy this dbt project to:
+
+  $ astro dbt deploy <deployment ID>
+
+Menu will be presented if you do not specify a deployment ID:
+
+  $ astro dbt deploy
+`
+
+	dbtDefaultMountPathPrefix = "/usr/local/airflow/dbt/"
+	dbtProjectYmlFilename     = "dbt_project.yml"
+	dbtBundleType             = "dbt"
+)
+
+func newDbtCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "dbt",
+		Short: "Manage your dbt projects deployed to Deployments running on Astronomer",
+	}
+	cmd.AddCommand(
+		newDbtDeployCmd(),
+	)
+	return cmd
+}
+
+func newDbtDeployCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "deploy DEPLOYMENT-ID",
+		Short:   "Deploy your dbt project to a Deployment on Astro",
+		Long:    "Deploy your dbt project to a Deployment on Astro. This command bundles your dbt project files and uploads it to your Deployment.",
+		Args:    cobra.MaximumNArgs(1),
+		RunE:    deployDbt,
+		Example: dbtDeployExample,
+	}
+
+	cmd.Flags().StringVarP(&mountPath, "mount-path", "m", "", "Path to mount dbt project in Airflow, for reference by DAGs. Default /usr/local/dbt/{dbt project name}")
+	cmd.Flags().StringVarP(&dbtProjectPath, "project-path", "b", "", "Path to the dbt project to deploy. Default current directory")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&dbtProjectPath, "project-path", "p", "", "Path to the dbt project to deploy. Default current directory")
```
nit: maybe a better short flag, or is it already taken up by something?

Reply:
Thanks, that was an oversight.